### PR TITLE
Feature/create messages

### PR DESCRIPTION
**Pull request by Ayokunnumi1**
These are the project requirement fulfilled in this branch;

Create a new React app called 'hello-react-front-end'
Initialize your project with Git.
Create the App component with react-router.
Create the Greeting component that will display a greeting. Set it up as a route in your App component.
Create a store, an action and a reducer that will connect to you API endpont to get the random greeting.
Display the random greeting in your Greeting component.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Provider } from 'react-redux';
+import store from './redux/store';
+import Greeting from '../src/components/Greeting';
+
+const App = () => (
+  <Provider store={store}>
+    <Greeting />
```
Comment (LionRouge1) [inline src/App.jsx:7]: - [ ] Kudos, You have done a great job displaying the greeting from the API endpoint :clap: However, according to the project requirement we should set up a route in the `App` component that leads to the Greeting component. Therefore I suggest that you make a use of **`react-route`** to create a link in the `App` component that link to the **`Greeting`** component to follow the project requirement.
Screenshot from 2024-02-02 22-49-31

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @Ayokunnumi1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- The two apps work together :heavy_check_mark: 
- Use redux to connect to the API endpoint :heavy_check_mark: 
- `App` component have route that leads to the `Greeting` component :x:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — COMMENTED** [review]
Hi @Ayokunnumi1 👋🏼👋🏼, 

approved


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅✅

- [x] User interface looks great👍🏼.
- [x] `react-router` has been used👍🏼.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ayokunnumi1** [comment]
I mistakenly merged the branch after the first reviewer approved the backend without waiting on his review of the frontend. I have implemented the changes without creating a new pull request

**Pull request merged**